platform/iphone/app_delegate.mm

```diff
@@ -665,6 +665,7 @@ - (void)applicationDidFinishLaunching:(UIApplication *)application {
 												  isAdvertisingTrackingEnabled]];
 
 #endif
+	return TRUE;
```

Comment:
There are some early returns above where you should return FALSE.

Reply:
I saw that, but it's actually dead code, there is no "game_analytics" module.
there are several unused module codes in here.

Reply:
It would be good to remove such dead code then :) iOS really needs a cleanup.

Reply:
Well, I was right, iOS fails building after this merge:
```
Compiling ==> platform/iphone/view_controller.mm
platform/iphone/app_delegate.mm:606:3: error: non-void method 'application:didFinishLaunchingWithOptions:' should return a value [-Wreturn-type,2]
                return;
                 ^
```

Reply:
weird. I had compiled successfully with this.

Reply:
Can you test #14497 ?